rust/lance-file/src/reader.rs

```diff
@@ -322,7 +326,7 @@ impl FileReader {
                         .await
                 }
             })
-            .buffered(num_cpus::get() * 4)
+            .buffered(self.io_parallelism() as usize)
```

Comment:
can `io_parallalism` return `usize` so dont need cast all of them?

Reply:
Good idea.  Done.